src/app/redux/store.ts

```diff
@@ -30,6 +31,8 @@ export const createAppStore = (options?: ConfigureStoreOptions['preloadedState']
 
 export const store = createAppStore();
 
+setupListeners(store.dispatch);
```

Comment:
Nice job finding this, looks like this enables rtk-query's focus and reconnect tracking.